src/dialogs/dropdownImpl.ts

```diff
@@ -0,0 +1,66 @@
+import { ComponentImpl } from "./componentImpl";
+import { DropDownComponent, ModelComponentTypes, ComponentEventType, CategoryValue} from "./interfaces";
+import { Emitter } from "vscode-languageclient";
+import * as vscode from 'vscode';
+
+export class DropDownWrapper extends ComponentImpl implements DropDownComponent {
```

Comment:
let's stick with the ...Impl naming convention instead of mixing with ...Wrapper